frontend/package.json

```diff
@@ -10,25 +10,30 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@tailwindcss/vite": "^4.1.13",
+    "@tanstack/react-router": "^1.131.44",
+    "@tanstack/react-router-devtools": "^1.131.44",
```

Comment:
nit: `@tailwindcss/vite` and `@tanstack/react-router-devtools` can probably be moved as dev deps